src/conmon.c

```diff
@@ -36,6 +36,13 @@ static void disconnect_std_streams(int dev_null_r, int dev_null_w)
 
 #define DEFAULT_UMASK 0022
 
+#ifdef __linux__
```

Comment:
typically, we check the inverse:
https://github.com/containers/conmon/pull/427/files
can we switch the case here?